walkthroughs/tls-with-acm/README.md

```diff
@@ -62,15 +62,39 @@ Finally, build and deploy the color app images.
 
 Before we can encrypt traffic between services in the mesh, we need to generate a certificate.
 
-To keep things simple, we'll use a single self-signed certificate for this walkthrough. However, you can also use certificates signed by an ACM Private Certificate Authority to enable TLS encryption in App Mesh. [See below](##extra-create-a-private-certificate-authority-and-certificate) for how to create a Private Certificate Authority and certificate.
+App Mesh currently supports two approaches to using certificates from ACM:
 
-This certificate will be used to encrypt traffic between the color teller and color gateway Virtual Nodes. In case you want to experiment further after this walkthrough, we'll create a wildcard certificate for the whole service domain, but we'll only be applying it to the Color Teller Virtual Node.
+1. [Imported certificates](https://docs.aws.amazon.com/acm/latest/userguide/import-certificate.html)
+2. Certificates issued by an [ACM Private Certificate Authority](https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaWelcome.html)
+
+To keep things simple, we'll use an imported certificate for this walkthrough. If you're interested in setting up a Private CA in ACM for this walkthrough, [see below](##extra-create-a-private-certificate-authority-and-certificate).
```

Comment:
I can't tell if that `see below` link would work since it's not getting highlighted as such. It should be a single... pound symbol... right?

Reply:
You're right. Interestingly it works in my local preview. I'll update it. :)